tests/foreman/cli/test_errata.py

```diff
@@ -459,6 +460,36 @@ def test_negative_install_by_hc_name_without_org_info(self):
             )
         self.assertIn('Error: Could not find organization', context.exception.stderr)
 
+    @tier3
+    @upgrade
+    def test_positive_list_affected_chosts(self):
+        """View a list of affected content hosts for an erratum
+
+        :id: 3b592253-52c0-4165-9a48-ba55287e9ee9
+
+        :Setup: Errata synced on satellite server.
+
+        :Steps: host list --search "applicable_errata = <erratum_id>"
+            --organization-id=<org_id>
+
+        :expectedresults: List of affected content hosts for an erratum is
+            displayed.
+
+        :CaseAutomation: automated
+
+        """
+        with ssh.get_connection() as connection:
+            result = connection.run(
+                'hammer host list --search "applicable_errata = {}" --organization-id={}'
+                ' --fields="Name"'.format(FAKE_2_ERRATA_ID, self.org['id'])
+            )
+            for virtual_machine in self.virtual_machines:
+                match = re.search(r'{}'.format(virtual_machine.hostname), str(result.stdout))
+                assert match, "VM host name not found in list of applicable hosts"
+            for virtual_machine in self.virtual_machines:
+                match = re.search(r'{}'.format(virtual_machine.hostname), str(result))
+                assert match, "VM host name not found in list of applicable hosts"
```

Comment:
What's the point? `str(result)` returns something like
```
"SSHCommandResult(stdout=['bundler', 'hsperfdata_puppet', 'hsperfdata_root', 'hsperfdata_tomcat', 'systemd-private-fcc9f585f1e4457a9e85e110168f218b-chronyd.service-H1UsiX', 'systemd-private-fcc9f585f1e4457a9e85e110168f218b-httpd.service-IBlRr0', 'systemd-private-fcc9f585f1e4457a9e85e110168f218b-rh-mongodb34-mongod.service-nlVId6', ''], stderr=b'', return_code=0, output_format=None)"
```
So, `result.stdout` and some more that we probably don't care about. Seems to me it very much repeats previous loop.

Reply:
Oh right, you mean just use 'result.stdout', OK.